<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">/* Main Window Styling */
QWidget {
    background-color: #444444;
    color: #ffffff;

}

/* Label Styling */
QLabel {
    background: transparent;
    color: #d3d3d3;
font-family: Arial, sans-serif;
    font-size: 12px;

}

/* ComboBox Styling */
QComboBox {
    background-color: #5D5D5D;
    color: #ffffff;
    border: 1px solid #5c5c5c;
    border-radius: 4px;
    padding: 2px 4px;

}
QComboBox QAbstractItemView {
    background-color: #3c3f41;
    selection-background-color: #4a90e2;
    selection-color: #ffffff;
    border: 1px solid #5c5c5c;
}

/* Button Styling */
QPushButton {
    background-color: #3c3f41;
    color: #ffffff;
    border: 1px solid #5c5c5c;
    border-radius: 4px;
    padding: 4px 8px;
}
QPushButton:hover {
    background-color: #4a4a4a;
}
QPushButton:pressed {
    background-color: #5a5a5a;
}

/* Line Edit Styling */
QLineEdit {
    background-color: #2B2B2B;
    color: #ffffff;
    border: 1px solid #5c5c5c;
    border-radius: 4px;
    padding: 2px 4px;
}



QDoubleSpinBox {
    background-color: #2B2B2B;
    color: #ffffff;
    border: 1px solid #5c5c5c;
    border-radius: 4px;
    padding: 2px 4px;
}

/* Hide the up and down buttons */
QDoubleSpinBox::up-button, QDoubleSpinBox::down-button {
    width: 0;
    height: 0;
    border: none;
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="verticalSpacing">
     <number>2</number>
    </property>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="verticalSpacing">
       <number>1</number>
      </property>
      <item row="1" column="0">
       <widget class="QComboBox" name="cmbChainShape">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial,sans-serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>ChainPipe</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="btnAddNewBaseShape">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">    background-color: #5D5D5D;</string>
        </property>
        <property name="text">
         <string>Add New Base Shape</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_92">
        <property name="font">
         <font>
          <family>Arial,sans-serif</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(199, 199, 199);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#d1d1d1;&quot;&gt;Select Chain Shape:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_94">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#d1d1d1;&quot;&gt;Chain Link Scale:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>83</number>
      </property>
      <property name="rightMargin">
       <number>240</number>
      </property>
      <property name="horizontalSpacing">
       <number>25</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_95">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ffffff;&quot;&gt;Scale (X Y Z)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="scaleX">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>45</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QDoubleSpinBox" name="ScaleY">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>45</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QDoubleSpinBox" name="ScaleZ">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>45</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="horizontalSpacing">
       <number>14</number>
      </property>
      <item row="0" column="1">
       <widget class="QSlider" name="horizontalSlider">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QSlider {
    background: transparent;
}

QSlider::groove:horizontal {
    height: 10px;
    background: #2B2B2B;
    border: 1px solid #3c3f41;
    border-radius: 3px;
}

QSlider::handle:horizontal {
    width: 12px;
    height: 20px; /* Height greater than groove */
    background: #d3d3d3;
    border: 1px solid #5c5c5c;
    border-radius: 8px; /* Rounded handle */
    margin: -9px 0; /* Centers the handle vertically over the groove */
}

QSlider::sub-page:horizontal {

}
</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Translation along z axis</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_96">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#d1d1d1;&quot;&gt;Number of Links:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="txtNumberOfLinks">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="btnCreateChain">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">    background-color: #5D5D5D;</string>
        </property>
        <property name="text">
         <string>Create Chain</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
